src/main/java/net/minestom/server/item/metadata/PotionMeta.java

```diff
@@ -98,7 +98,7 @@ public void read(@NotNull NBTCompound nbtCompound) {
                 for (NBTCompound potionCompound : customEffectList) {
                     final byte id = potionCompound.getAsByte("Id");
                     final byte amplifier = potionCompound.getAsByte("Amplifier");
-                    final int duration = potionCompound.containsKey("Duration") ? potionCompound.getNumber("Duration").intValue() : (int) TimeUnit.SECOND.toMilliseconds(30);
+                    final int duration = potionCompound.containsKey("Duration") ? potionCompound.getNumber("Duration").intValue() : (int) TimeUnit.getMillis(30, TimeUnit.SECOND);
```

Comment:
We should probably be using `ChronoUnit` instead of the concurrent-based `TimeUnit`

Reply:
Well I used it because it's much shorter than writing `ChronoUnit.SECONDS.getDuration().multipliedBy(30).toMillis()` also i didn't want to mix the two like `TimeUnit.getMillis(30, ChronoUnit.SECONDS)`.